src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Adapter\ExtLdap;
+
+use Symfony\Component\Ldap\Adapter\AbstractConnection;
+use Symfony\Component\Ldap\Exception\ConnectionException;
+
+/**
+ * @author Charles Sarrazin <charles@sarraz.in>
+ */
+class Connection extends AbstractConnection
+{
+    /** @var bool */
+    private $bound = false;
+
+    /** @var resource */
+    private $connection;
+
+    public function __destruct()
```

Comment:
Yes. We want the linked resources to be cleared if someone explicitely wants to delete the object (i.e. using the Ldap component outside the Symfony framework).